packages/react-native-oursky/src/SignupWithMobile.js

```diff
@@ -150,13 +150,21 @@ export default class SignupWithMobile extends React.PureComponent<
 
   onChangeText = (text: string) => {
     if (/^\d*$/.test(text)) {
+      const { mobileNumberProps } = this.props;
+      if (mobileNumberProps && mobileNumberProps.onChangeText) {
```

Comment:
I found in the `SignupWithMobile` also have excessive null checking code which hurt my reading and introduce endless bugs on missed checking.

Can we make it non optional and give a default value?